theia-full-docker/Dockerfile

```diff
@@ -106,10 +106,22 @@ RUN echo "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-6.0 main" > /etc
 RUN apt-get update && apt-get install -y clang-tools-6.0
 RUN ln -s /usr/bin/clangd-6.0 /usr/bin/clangd
 
-#Python
-RUN apt-get update && apt-get install -y python python-pip
-RUN pip install python-language-server
-
+#Python 2
+RUN apt-get update && apt-get install -y python python-pip && \
+ pip install python-language-server;
+    
+#Developer tools
+
+## Git and sudo (sudo needed for user override)
+RUN sudo apt-get -y install git sudo
```

Comment:
I get the following error: ```/bin/sh: 1: sudo: not found```
I think the first ```sudo``` should be removed.

Reply:
Whoops, probably a small oversight, will fix